packages/grafana-ui/src/types/icon.ts

```diff
@@ -232,4 +233,5 @@ export const getAvailableIcons = (): IconName[] => [
   'cloud',
   'draggabledots',
   'folder-upload',
+  'clipboard-alt',
```

Comment:
```suggestion
```

I think we can just use the copy icon that we already have and use for most copy/copy to clipboard stuff.